docs/src/architecture/accounts.md

```diff
@@ -1,59 +1,61 @@
 
-* Accounts with **public state**, where the actual state is stored on chain. These would be similar to how accounts work in public blockchains.
-* Accounts with **private state**, where only the hash of the account is stored on chain. The hash is defined as: \
-`hash([account ID, 0, 0, nonce], [vault root], [storage root], [code root])`
+3. Fungible asset faucet with immutable code
+    Assets need to be created by accounts in Miden. The Account ID will start with `10`. 
 
-### Example account ID
+4. Non-fungible asset faucet with immutable code
+    Assets need to be created by accounts in Miden. The Account ID will start with `11`.
 
-<p align="center">
-    <img src="../diagrams/architecture/account/Account_ID.png">
-</p>
+### Account Storage Modes
+Account data which is stored in a Miden Node can be either `public` or `private`. The third most significant bit ([MSb](https://en.wikipedia.org/wiki/Bit_numbering)) of the Account ID specifies whether the account data is public (`0`) or private (`1`).
+
+0. Accounts with **public state** store their actual state on chain, similar to how accounts work in public blockchains like Ethereum.
+1. Accounts with **private state** store only the hash of their account on chain. The hash is defined as: \
+`hash([account ID, 0, 0, nonce], [vault root], [storage root], [code root])`
 
 ## Storage
-User-defined data that can be stored in an account. Account storage is composed of two components. The first component is a simple sparse Merkle tree of depth 8 which is index addressable. This provides the user with 256 Word slots. Users that require additional storage can use the second component which is a `MerkleStore`. This will allow the user to store any Merkle structures they need. This is achieved by storing the root of the Merkle structure as a leaf in the simple sparse merkle tree. When `AccountStorage` is serialized it will check to see if any of the leafs in the simple sparse Merkle tree are Merkle roots of other Merkle structures. If any Merkle roots are found then the Merkle structures will be persisted in the `AccountStorage` `MerkleStore`.
+Storage refers to `AccountStorage`, the user-defined data that can be stored in an account. Account storage is made up of two components. The first component is an index addressable basic Sparse Merkle Tree of depth 8. This gives the user 256-word slots.
+
+Users that require additional storage can use the second component which is a `MerkleStore`. This will allow the user to store any Merkle structures they need. This is achieved by storing the Merkle structure's root as a leaf in the simple sparse merkle tree. When `AccountStorage` is serialized, it will examine the leaves of the simple sparse Merkle tree to see whether any of them are Merkle roots of other Merkle structures. If any Merkle roots are found, the Merkle structures will be saved in the `AccountStorage` `MerkleStore`.
```

Comment:
```suggestion
Users that require additional storage can use the second component which is a `MerkleStore`. This will allow the user to store any Merkle structures they need. This is achieved by storing the Merkle structure's root as a leaf in the simple Sparse Merkle Tree. When `AccountStorage` is serialized, it will examine the leaves of the simple Sparse Merkle Tree to see whether any of them are Merkle roots of other Merkle structures. If any Merkle roots are found, the Merkle structures will be saved in the `AccountStorage` `MerkleStore`.
```